tests/test_functional.py

```diff
@@ -168,10 +168,8 @@ def test_obal_scratch_with_koji_upstream_hello():
     expected_log = [
         "koji build obaltest-nightly-rhel7 /tmp/SRPMs/hello-2.10-2.src.rpm --scratch",
         "koji watch-task 1234",
-        "koji taskinfo -v 1234",
```

Comment:
on the one hand, this seems unexpected.

on the other, we don't use `koji build` anywhwere anymore, right?
it's either `copr build` or `tito`?